sdk/aqueduct/tests/decorator_test.py

```diff
@@ -55,6 +55,14 @@ def test_handle_reserved_file_dependencies():
         )
 
 
+def test_missing_requirements_file():
+    with pytest.raises(FileNotFoundError):
+        _package_files_and_requirements(
+            python_function,
+            dir_path=os.path.join(os.getcwd(), "test_files"),
+        )
```

Comment:
Why does this fail? Isn't there a requirements.txt file in `test_files/python_function`? I guess I'm confused as to why there's a requirements.txt file in test_files/python_function at all.